catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/BasicTypes.java

```diff
@@ -311,7 +328,36 @@ public AttributeFormat getAttributeFormat() {
                 false /* tokenized */,
                 true /* multivalued */,
                 STRING_TYPE));
-        return descriptors;
+        DESCRIPTORS = Collections.unmodifiableSet(descriptors);
+
+        BASIC_METACARD = new MetacardTypeImpl(MetacardType.DEFAULT_METACARD_TYPE_NAME,
```

Comment:
I'm not sure we're talking about the same thing. Look at how the MetacardType is being composited in this PR: https://github.com/glenhein/ddf/blob/beed46142680a63d29927fce10285c557ada9e0a/catalog/transformer/catalog-transformer-pdf/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L22 You can't do that with a static constant. Besides, since all the fields are static and unmodifiable there's practically no cost associated with instantiating these classes.